<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QOmgWeaponsDialog</class>
 <widget class="QDialog" name="QOmgWeaponsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>357</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="_name"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="_pb_textures">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Select a Texture</string>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/Media/gun.png</normaloff>:/Media/gun.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Weapon Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbTypeWeapon">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Simple Shot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blast Shot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Multi Shot</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="bulletIcon">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../images.qrc">:/Media/bullet.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bulletLabel">
       <property name="text">
        <string>Bullets:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="bulletSpin">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="bulletIcon_2">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../images.qrc">:/Media/damage.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bulletLabel_2">
       <property name="text">
        <string>Damage</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="_damageSpin">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QColorButton" name="widget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../images.qrc">:/Media/folder.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Select a Folder..</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="folderList">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="_newWeapon">
         <property name="minimumSize">
          <size>
           <width>82</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Add Weapon</string>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/Media/add.png</normaloff>:/Media/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_editWeapon">
         <property name="minimumSize">
          <size>
           <width>82</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Edit Weapon</string>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/Media/edit.png</normaloff>:/Media/edit.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="_cancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/Media/delete.png</normaloff>:/Media/delete.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QColorButton</class>
   <extends>QWidget</extends>
   <header>QColorButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
